examples/src/main/java/com/mapbox/navigation/examples/util/RouteDrawingActivity.kt

```diff
@@ -79,13 +79,12 @@ class RouteDrawingActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.layout_route_drawing_activity)
-        val tileStore = TileStore.getInstance()
+        val tileStore = TileStore.create()
         val mapboxMapOptions = MapInitOptions(this)
         val resourceOptions = ResourceOptions.Builder()
             .accessToken(getMapboxAccessTokenFromResources())
             .assetPath(filesDir.absolutePath)
-            .cachePath(filesDir.absolutePath + "/mbx.db")
-            .cacheSize(100000000L) // 100 MB
```

Comment:
wasn't certain about this one. but it's only an example so seems right

Reply:
oh it says, so this is good

"Remove ResourceOptions::cacheSize and DefaultAmbientCacheSize constant. (https://github.com/mapbox/mapbox-maps-android/pull/399)"